server/index.js

```diff
@@ -25,7 +25,8 @@ if (process.env.DEV) {
     require('webpack-dev-middleware')(compiler, {
       noInfo: true,
       publicPath: webpackConfig.output.publicPath,
-      inline: true
+      inline: true,
+      logLevel: 'warn'
```

Comment:
Without this, Webpack is spitting out logs about 100+ tiny bundles being completed. Verify that...that's ok. It seems awfully weird, but source maps and everything are working perfectly well. The bundle sizes (in Kb's) seem like they would add up to about the same size as the old, single 4Mb-ish bundle.

Might just be a weird quirk of using `import()` dynamic imports 🤷‍♂️